src/main/scala/beam/physsim/bprsim/Coordinator.scala

```diff
@@ -52,9 +54,9 @@ class Coordinator(
   private def executePeriod(tillTime: Double): Unit = {
     val events = executeSubPeriod(tillTime, Vector.empty[Event])
     asyncFlushEvents(events)
-    val minTime = workers.map(_.minTime).min
-    if (!minTime.equals(Double.MaxValue)) {
-      executePeriod(minTime + config.syncInterval)
+    val minTime: Option[Double] = SequenceUtils.minOption(workers.map(_.minTime))
+    if (!minTime.contains(Double.MaxValue)) {
+      executePeriod(minTime.get + config.syncInterval)
```

Comment:
Won't this result in another Option.get error in Scapegoat?

Reply:
in this case will be solved during in the PR that fixes Options on scapegoat. At this moment I am trying to keep the same behaviour